tests/conftest.py

```diff
@@ -50,6 +50,9 @@ def pytest_configure():
     pytest.data_path_swl_project_file_in_container = (
         "C:\\data\\archive_geneSoundPowerLevelProject_hemisphere_2025R1_20243008.spw"
```

Comment:
@anshlachamb can you remove 'archive_gene' from the file name? I had removed it, it was added again apparently when the version/date suffix was added. Also, I think it would be best to have consistent naming for the two cases (calib and not calib)